test/templates/layout.njk

```diff
@@ -60,7 +60,7 @@ angular_material/1.1.4</label>
 
 
     </header>
-    <main style="background-color: white;">
+    <main class="fd-has-color-background">
```

Comment:
Shouldn't this be `fd-has-background-color-background` not `fd-has-color-background`. The latter changes the text color on the page.